client/render-system.cpp

```diff
@@ -217,7 +218,7 @@ namespace tec {
 
 		glBindVertexArray(this->sphere_vbo.GetVAO());
 		glBindBuffer(GL_ELEMENT_ARRAY_BUFFER, this->sphere_vbo.GetIBO());
-		std::size_t index_count = this->sphere_vbo.GetVertexGroup(0)->index_count;
+		auto index_count{ static_cast<GLsizei>(this->sphere_vbo.GetVertexGroup(0)->index_count) };
```

Comment:
This is really ugly compared to what was there before. I assume it had a conversion warning or something like that?

Reply:
yes